Activating extension `lukehoban.Go` failed: Cannot find module '/Users/jonathan/vscode-go/out/src/goMain'.

Following the building and debugging instructions, when I open go source files in the Exntesion Development Host I get this:

```
Activating extension `lukehoban.Go` failed:  Cannot find module '/Users/jonathan/vscode-go/out/src/goMain'
Here is the error stack:  Error: Cannot find module '/Users/jonathan/vscode-go/out/src/goMain'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Function.n._load (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/pluginHostProcess.js:18:17333)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Function.t [as __$__nodeRequire] (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js:5:1298)
    at u (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/pluginHostProcess.js:12:20550)
    at t._loadPluginModule (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/pluginHostProcess.js:12:24382)
    at t.e._actualActivatePlugin (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/pluginHostProcess.js:11:29061)
    at t._actualActivatePlugin (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/node/pluginHostProcess.js:12:24874)

```

```
module.js:338
    throw err;
          ^
Error: Cannot find module 'typescript'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.require.resolve (module.js:388:19)
    at Object.<anonymous> (/Users/jonathan/vscode-go/node_modules/vscode/bin/compile:4:40)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

--- a/package.json
+++ b/package.json
@@ -34,7 +34,8 @@
 	},
 	"devDependencies": {
 		"vscode": "0.10.x",
-		"fs-extra": "^0.26.0"
+		"fs-extra": "^0.26.0",
+		"typescript": "^1.6.2"
 	},
 	"engines": {
 		"vscode": "0.10.x"

